tests/test_openapi.py

```diff
@@ -45,3 +58,602 @@ def test_str2bool():
 
     default['ogc_schemas_location'] = '/opt/schemas.opengis.net'
     osl = get_ogc_schemas_location(default)
+
+
+def get_test_file_path(filename):
+    """
+    Helper function to open test file safely
+
+    :param filename: file path
+
+    :returns: corrected file path
+    """
+
+    if os.path.isfile(filename):
+        return filename
+    return 'tests/{}'.format(filename)
+
+
+@pytest.fixture()
+def config():
+    """
+    Get the OpenAPI Document configuration
+
+    :returns: OpenAPI configuration YAML dict
+    """
+    with open(get_test_file_path('pygeoapi-test-openapi-config.yml')) as config_file:  # noqa
+        return yaml_load(config_file)
+
+
+@pytest.fixture()
+def get_oas_30_(config):
```

Comment:
Isn’t better to qualify the function name without the verb as it is a fixture?

Reply:
removed from all